vividus-to-xray-exporter/src/main/java/org/vividus/xray/factory/TestExecutionFactory.java

```diff
@@ -89,4 +96,11 @@ private static TestExecutionItemStatus calculateStatus(AbstractStepsContainer st
                      .allMatch(step -> "successful".equals(step.getOutcome())) ? TestExecutionItemStatus.PASS
                              : TestExecutionItemStatus.FAIL;
     }
+
+    private static String asOffsetDateTime(long nanos)
+    {
+        long millis = TimeUnit.MICROSECONDS.convert(nanos, TimeUnit.NANOSECONDS);
+        Instant instant = Instant.ofEpochSecond(millis);
```

Comment:
something wrong in 2 above lines: microseconds treated as milliseconds, milliseconds treated as seconds

Reply:
confused nanos with millis, fixed